src/ImageSharp/Formats/Jpeg/Components/Decoder/JpegFrame.cs

```diff
@@ -86,7 +86,7 @@ public JpegFrame(JpegFileMarker sofMarker, byte precision, int width, int height
         /// <summary>
         /// Gets or sets the frame component collection.
         /// </summary>
-        public JpegComponent[] Components { get; set; }
+        public IJpegComponent[] Components { get; set; }
```

Comment:
As far as I remember, `IJpegComponent` interface is used in tests for libjpeg output comparison but I think we shouldn't use this interface everywhere - [this leads to virtual calls for every component field getter ](https://sharplab.io/#v2:EYLgHgbALANALiAhgZwLYB8ACAmAjAWAChMBmAAk1wguzIGEyBvIs1i8gSwDs4BTAJwBmiAMa8yASToB7VAAdpXXjzIs2zQmy1lg06QBsyAMX2IA5kzO84AbgC+a1g8KP2NerIVKVISTPmKynCuGtpspDp6hibmltb2rs5arhGU1LoGZADi1jFmABRSnoEqIp4AlGQAvAB8ZGXyAHR5NinkaZGZOXB5+f5eQfUV1XUNcs2mZq2EdkA==).